tests/test_app/urls.py

```diff
@@ -1,6 +1,6 @@
-from django.urls import path
+from django.urls import re_path
 
 from .views import index
 
 app_name = "tests.test_app.views"
-urlpatterns = [path("", index, name="index")]
```

Comment:
I believe `path` is OK here. `re_path` is just the replacement for `url` if you didn't want to use the new `path`

Reply:
Too late

Reply:
Yeah, you are correct @jtrain - I've gotten a bit overzealous :D.